src/manifest.json

```diff
@@ -56,7 +58,8 @@
         "https://*.ses.nsw.gov.au/Messages/Create",
         "https://*.ses.nsw.gov.au/Messages/Create/"
       ],
-      "js": ["contentscripts/messages/create.js"]
+      "js": ["contentscripts/messages/create.js"],
+      "run_at" : "document_end"
```

Comment:
Rather than this just wrap the sensitive code (most likely just the last line) in jQuery.ready:

    $(function() {
        // things to do when page is ready
    });